### PR TITLE
Several fixes, among which Abstract reviewing improvements

### PR DESCRIPTION
**Pull request by pferreir**
Supersedes #2377

### CODE REVIEW

File: indico/modules/events/templates/management/_contribution_field_list.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
             <div class="contribution-field">
                 <span class="field-title">{{ field.title }}</span>
                 {% if field.field_type %}
-                    <span class="field-type">({{ field.field.friendly_name }})</span>
+                    <span class="field-type">({{ field.get_field().friendly_name }})</span>
```
Comment (ThiefMaster) [inline indico/modules/events/templates/management/_contribution_field_list.html]: Wouldn't it be more consistent to use `management=True` here even though it makes no difference at the moment?

File: indico/modules/events/timetable/legacy.py [hunk 2]
```diff
@@ -201,7 +201,7 @@ def _get_date_data(self, entry):
         if self.management:
             tzinfo = entry.event_new.tzinfo
         else:
-            tzinfo = timezone(entry.event_new.as_legacy.tz)
+            tzinfo = timezone(entry.event_new.display_tzinfo)
```
Comment (ThiefMaster) [inline indico/modules/events/timetable/legacy.py]: no need for the `timezone()` call, it's already done in `display_tzinfo`

File: indico/modules/events/templates/management/contribution_field_list.html [hunk 3]
```diff
@@ -4,6 +4,6 @@
 (<a href="#" class="js-dialog-action"
     data-title="{% trans %}Manage fields{% endtrans %}"
     data-href="{{ url_for('contributions.manage_fields', event) }}"
-    data-ajax-dialog>
+    data-ajax-dialog data-reload-after="customData">
```
Comment (ThiefMaster) [inline indico/modules/events/templates/management/contribution_field_list.html]: I'd put it on its own line

File: indico/modules/events/contributions/models/fields.py [hunk 4]
```diff
@@ -99,14 +99,13 @@ class ContributionField(db.Model):
     # - abstract_values (AbstractFieldValue.contribution_field)
     # - contribution_values (ContributionFieldValue.contribution_field)
 
-    @property
-    def field(self):
+    def get_field(self, management=False):
```
Comment (ThiefMaster) [inline indico/modules/events/contributions/models/fields.py]: I wonder if it'd make more sense to have `field` and `mgmt_field` (or `management_field`, but that's a bit long ;)) properties instead (which then call `_get_field(management=True|False)`)

File: indico/modules/events/timetable/legacy.py [hunk 5]
```diff
@@ -196,7 +198,10 @@ def _get_color_data(self, obj):
                 'textColor': '#' + obj.text_color}
 
     def _get_date_data(self, entry):
-        tzinfo = entry.event_new.tzinfo
+        if self.management:
+            tzinfo = entry.event_new.tzinfo
+        else:
+            tzinfo = timezone(entry.event_new.display_tzinfo)
```
Comment (ThiefMaster) [inline indico/modules/events/timetable/legacy.py]: timezone() is done by `display_tzinfo`

File: indico/modules/events/sessions/templates/management/session_blocks.html [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+            <th class="i-table">{% trans %}Start date{% endtrans %}</th>
+            <th class="i-table">{% trans %}Duration{% endtrans %}</th>
+            <th class="i-table">{% trans %}Contributions{% endtrans %}</th>
+        </tr>
+    </thead>
+    <tbody>
+        {% for block in sess.blocks %}
+            <tr class="i-table">
+                <td class="i-table">{{ block.title }}</td>
+                <td class="i-table">{{ block.start_dt|format_date }}</td>
```
Comment (ThiefMaster) [inline indico/modules/events/sessions/templates/management/session_blocks.html]: `format_datetime`?

File: indico/modules/events/templates/management/_contribution_field_list.html (hunk 1 above)
Comment (pferreir) [inline indico/modules/events/templates/management/_contribution_field_list.html]: Thought of it, realized it makes no difference... 😛 
But yeah, you're right.

File: indico/modules/events/contributions/models/fields.py (hunk 4 above)
Comment (pferreir) [inline indico/modules/events/contributions/models/fields.py]: Seems a bit cryptic to me. TBH, the current solution already seems pretty exaggerated to me. I cannot think of anything better, however.

File: indico/modules/events/timetable/legacy.py (hunk 5 above)
Comment (pferreir) [inline indico/modules/events/timetable/legacy.py]: ah true. should have tested that 😕

File: indico/modules/events/contributions/models/fields.py (hunk 4 above)
Comment (pferreir) [inline indico/modules/events/contributions/models/fields.py]: Yeah, OK, maybe your solution is indeed clearer.

**Pull request merged**